protostar/testing/test_scheduler.py

```diff
@@ -33,10 +77,12 @@ def run(
         exit_first: bool,
         testing_seed: Seed,
         max_steps: Optional[int],
-        project_root_path_str: str,
+        project_root_path: Path,
         cwd: Path,
         active_profile_name: Optional[str],
         gas_estimation_enabled: bool,
+        testing_summary: TestingSummary,
+        json_format: bool = False,
```

Comment:
`TestScheduler` shouldn't know the format (and logging in general). TestScheduler and LiveLogger need to be refactored before adding JSON support.

Reply:
Can you specify what refactoring you mean? Is there a task for this? Are you sure that refactoring cannot be applied after this PR is merged?

Reply:
Nvm. It can be postponed. I thought `LiveLogger` is in the protostar/testing module and that would cause cyclic dependency problems.